src/plugins/saved_objects_management/public/management_section/recent_work.tsx

```diff
@@ -115,51 +126,143 @@ export const RecentWork = (props: { core: CoreStart; workspaceEnabled?: boolean
     return options.map((option: string) => ({ label: option, value: option }));
   }, [detailedSavedObjects]);
 
-  const capitalTheFirstLetter = function (recentAccessItem: DetailedRecentlyAccessedItem) {
-    return recentAccessItem.type.charAt(0).toUpperCase() + recentAccessItem.type.slice(1);
+  const capitalTheFirstLetter = function (str: string) {
+    return str ? str.charAt(0).toUpperCase() + str.slice(1) : '';
   };
 
   const itemsForDisplay = useMemo(() => {
-    const sortedResult = [...detailedSavedObjects]
-      .filter((item) => !item.error)
-      .sort(sortBy(sortKeyMap[selectedSort]));
-    return sortedResult.filter((item: SavedObject & ChromeRecentlyAccessedHistoryItem) => {
+    const sortedResult = [...detailedSavedObjects].filter((item) => !item.error);
```

Comment:
Shall we rename `sortedResult`? seem it doesn't sort it here any more. Also I think we can combine the two `filter`

Reply:
yes, combine them together make more sense.